onboard/macos/cassandra.md

```diff
@@ -0,0 +1,19 @@
+# Temporal Server Cassandra Setup Guide
+
+## v3.11
+
+### Install
+```zsh
```

Comment:
why not:
```suggestion
```bash
```
as everywhere else?

Reply:
what's the difference?

Reply:
At least in our own docs site, zsh syntax highlighting is not supported.

So for consistency, and in case these files get opened in something that supports syntax highlighting, we have been using bash -